App/Screens/ShareScreen/ShareScreen.tsx

```diff
@@ -15,14 +15,15 @@
 // along with Sh**t! I Smoke.  If not, see <http://www.gnu.org/licenses/>.
 
 import * as Sharing from 'expo-sharing';
-import React, { createRef, useEffect } from 'react';
+import React, { createRef, useEffect, useContext } from 'react';
```

Comment:
```suggestion
import React, { createRef, useContext, useEffect } from 'react';
```

Reply:
Sure, if it break your codestyle I will fix it